tests/xfstests/install.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Install xfstests
+# Maintainer: Nathan Zhao <jtzhao@suse.com>
+package install;
+
+use 5.018;
+use strict;
+use warnings;
+use base "opensusebasetest";
+use utils qw(zypper_call);
+use testapi;
+
+my $LOG_FILE = "/tmp/xfstests.log";
+
+# Create log file used to generate junit xml report
+sub log_create {
+    my $file = shift;
+    my $cmd = "[[ -f $file ]] || ";
+    $cmd .= "echo 'Test in progress' > $file";
+    assert_script_run($cmd);
+}
+
+sub run {
+    my $self = shift;
+    select_console('root-console');
```

Comment:
@jtzhao As I wrote before, please use `select_virtio_console()` instead of `select_console('root-console')`. This will allow use virtio terminal, if you run test with `VIRTIO_CONSOLE=1`.

Using virtio console for tests it's much faster (see https://openqa.opensuse.org/tests/661009, output is in `serial_terminal.txt` log file: https://openqa.opensuse.org/tests/661009/file/serial_terminal.txt or in `serial0.txt` for some archs).

Virtio console was written by Richard Palethorpe for LTP tests, but it has been adopted for other tests (Anton is using for wicked tests, Jose for OPNFV tests, ...).

Reply:
This test won't run properly with virtio console.

Reply:
@jtzhao Why?

Reply:
@pevik Due to different behavior of wait_serial.